src/index.js

```diff
@@ -7,6 +7,7 @@ export { default as Button } from './elements/Button/Button';
 export { default as Input } from './elements/Input/Input';
 export { default as Text } from './elements/Text/Text';
 export { default as List } from './elements/List/List';
-
+export { default as TabbedContainer } from './elements/TabbedContainer/TabbedContainer';
+export { default as Tab } from './elements/TabbedContainer/Tab';
```

Comment:
Let's move the `Tab` component to its own directory. Even though a `Tab` won't be used outside of the context of a tabbed container, keeping everything flattened makes it easy to know where components are